### PR TITLE
Update documentation

### LINKED ISSUE #327: Update documentation to match new syntax

### PR DESCRIPTION
**Pull request by bqpd**
Closes #327

### CODE REVIEW

**Comment by whoburg** [comment]
Love it, looks great

**Comment by whoburg** [comment]
Not merging because the commit comment "begin updating docs" makes it seem like there is more coming.

**Comment by bqpd** [comment]
Yupyup.

**Comment by bqpd** [comment]
Alright, this is the last step!

File: docs/source/advancedcommands.rst [hunk 1]
```diff
@@ -1,6 +1,35 @@
 Advanced Commands
 *****************
 
+Feasibility Analysis
+====================
+
+If your Model doesn't solve, you can automatically find the nearest feasible version of it with the ``Model.feasibility()`` command, as shown below. The feasible version can either involve dividing all constraints by the smallest number possible, dividing each constraint by its own number and minimizing the product of those numbers, or changing each constant by the smallest total percentage possible.
+
```
Comment (whoburg) [inline docs/source/advancedcommands.rst]: instead of "dividing constraints by", maybe "relaxing constraints by"?

Comment (bqpd) [inline docs/source/advancedcommands.rst]: I like that, but I think we should also mention what that means; not all users will know that the whole constraint ends up on the left-hand side. ("dividing by" also fails to mention this, whoops. so maybe "relaxing" is better?)

Comment (whoburg) [inline docs/source/advancedcommands.rst]: sure, "relaxing" is fine by me!

**Comment by whoburg** [comment]
this looks great! @bqpd please go ahead and merge when you are finished pushing any changes.

File: docs/source/advancedcommands.rst (hunk 1 above)
Comment (bqpd) [inline docs/source/advancedcommands.rst]: updated!

**Pull request merged**